src/parser/parser.yy

```diff
@@ -2609,6 +2606,45 @@ comment_prop
     }
     ;
 
+opt_index_param_list
+    : %empty {
+        $$ = nullptr;
+    }
+    | index_param_list {
+        $$ = $1;
+    }
+    ;
+
+index_param_list
+    : index_param_item {
+        $$ = new IndexParamList();
+        $$->add($1);
+    }
+    | index_param_list COMMA index_param_item {
+        $$ = $1;
+        $$->add($3);
+    }
+    ;
+
+index_param_item
+    : comment_prop_assignment {
+        $$ = new IndexParamItem(IndexParamItem::COMMENT, *$1);
+    }
+    | KW_S2_MAX_LEVEL ASSIGN legal_integer {
```

Comment:
Why don't use label?

Reply:
And you must add incompatible when add new reserved word.

Reply:
> Why don't use label?

Do you mean use LABEL to replace `KW_S2_MAX_LEVEL`? I have considered it, but our syntax already use the exact name in parser.yy, like `TTL`, `COMMENT`.

Reply:
> And you must add incompatible when add new reserved word.

Well, strictly speaking, it does cause a little incompatibility.

Reply:
`COMMENT` is same as `WHERE` or other keyword to identify a part of statement. But `s2_max_level` is a property of part of statement.